### PR TITLE
Don't use head repository for non-head builds

### PR DESCRIPTION
**Pull request by martijnbastiaan**

### CODE REVIEW

File: cabal.project [hunk 1]
```diff
@@ -19,15 +19,6 @@ allow-newer:
   , rewrite-inspector:hashable
   , brick:base
 
-repository head.hackage.ghc.haskell.org
```
Comment (christiaanb) [inline cabal.project]: We sadly need this for building locally with GHC 8.8.1 though

Comment (martijnbastiaan) [reply cabal.project]: Just for `vty`/`rewrite-inspector` right?

Comment (christiaanb) [reply cabal.project]: I guess?

Comment (martijnbastiaan) [reply cabal.project]: Hopefully fixed in https://github.com/omelkonian/rewrite-inspector/pull/3 :-)

**Comment by martijnbastiaan** [comment]
@christiaanb ACK?

**Pull request merged**